packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/colorgrid/colorgrid.css

```diff
@@ -20,47 +21,68 @@
 }
 
 .ck.ck-color-grid__tile {
-	width: var(--ck-color-grid-tile-size);
-	height: var(--ck-color-grid-tile-size);
-	min-width: var(--ck-color-grid-tile-size);
-	min-height: var(--ck-color-grid-tile-size);
-	padding: 0;
 	transition: .2s ease box-shadow;
```

Comment:
I understand that this applies to both normal and contrast mode. Do we want to keep transitions for contrast mode?

Reply:
I think these two domains of UX are orthogonal. I see nothing wrong with leaving it as it is.